app/src/main/java/ai/elimu/vitabu/ui/storybook/WordViewAdapter.java

```diff
@@ -112,26 +113,40 @@ public EmptyViewHolder(@NonNull View itemView) {
     public static class WordViewHolder extends RecyclerView.ViewHolder {
 
         private final TextView wordText;
+        private final TextView wordEmoji;
         private final View wordUnderline;
 
         public WordViewHolder(@NonNull View itemView) {
             super(itemView);
             wordText = itemView.findViewById(R.id.word_text);
             wordUnderline = itemView.findViewById(R.id.word_underline);
+            wordEmoji = itemView.findViewById(R.id.word_emoji);
         }
 
         public void paintWordLayout(String word, WordGson wordWithAudio, int readingLevelPosition) {
```

Comment:
Perhaps rename the String from `word` to `wordText`, and the TextView from `wordText` to `wordTextView` to avoid confusion between those two?

```suggestion
        public void paintWordLayout(String wordText, WordGson wordWithAudio, int readingLevelPosition) {
```